Updated readme (#158)
gitmedha/sis-backend: README.md

# Strapi application
### Pre-Requisites

A quick description of your strapi application

## Project setup
```
yarn install
    1. Node :  ">=10.16.0 <=14.x.x",
    2. Postgres : version 14
```

### Compiles and starts strapi for development
```
yarn develop
```
### After compeleting installation of above mentioned pre-requisites, follow these steps :

### Compiles strapi for for production
```

> yarn build

Reviewer: I think, we need to add this also.

> * Create a postgres database `medha_sis`.

Reviewer: ```suggestion
* Create a PostgreSQL database named `medha_sis`.
```

> * Download this database dump fro provided link here: https://drive.google.com/drive/folders/1lSvMJr8AMiD68zqcpr2w0BnZ5N80EzE5

Reviewer: ```suggestion
* Download this database dump from the provided link here: https://drive.google.com/drive/folders/1lSvMJr8AMiD68zqcpr2w0BnZ5N80EzE5
```

> 3. Once database is restored run `yarn develop` and you will be able to access backend at `http://localhost:1337/`

Reviewer: ```suggestion
3. Once the database is restored, run `yarn develop`, and you will be able to access the backend at `http://localhost:1337/`.
```
